docs/learning/backend/CodeService.md

```diff
@@ -2,7 +2,7 @@
 
 ## Codes Overview
 
-[iTwin.js](https://www.itwinjs.org) is a platform for infrastructure digital twins that provides techniques for creating digital models of real world assets such that the physical world and the digital world are *connected*. In iTwin.js, digital models are persisted in [iModels](https://www.itwinjs.org/learning/glossary/#imodel), according to the rules of [BIS](https://www.itwinjs.org/bis/). [Codes](https://www.itwinjs.org/bis/intro/codes/) play an integral role in forming the connection between the physical world and the digital world.
+[iTwin.js](https://www.itwinjs.org) is a platform for infrastructure digital twins that provides techniques for creating digital models of real world assets such that the physical world and the digital world are *connected*. In iTwin.js, digital models are persisted in [iModels](https://www.itwinjs.org/learning/glossary/#imodel), according to the rules of [BIS](https://www.itwinjs.org/bis/). [Codes](https://www.itwinjs.org/bis/guide/fundamentals/codes/) play an integral role in forming the connection between the physical world and the digital world.
```

Comment:
This is a very informative document. Can you link to it from the API documentation, e.g., in `CodeService.ts`? I could find no existing links to it.

Reply:
done